.github/workflows/main.yml

```diff
@@ -51,3 +51,10 @@ jobs:
       - name: Run tests
         run: npm test
         working-directory: server/
+
+      - name: Linting
```

Comment:
Please make this a separate job, possibly called Linting or something like that.

We don't want to attach jobs with different functionalities to the same job! This makes it easier to "fix" when it throws a failed test!

Reply:
Makes a lot of sense, there are a lot of steps in that job to begin with. I'll shift some of the other steps into seperate jobs as well.